apps/system/test/unit/task_manager_test.js

```diff
@@ -1263,4 +1265,20 @@ suite('system/TaskManager >', function() {
     });
 
   });
+
+  suite('orientation', function() {
+    var app;
+    setup(function() {
+      app = MockService.currentApp = apps['http://sms.gaiamobile.org'];
+      MockStackManager.mCurrent = 0;
+    });
+
+    test('lock orientation when showing', function() {
+      this.sinon.stub(screen, 'mozLockOrientation');
+      showTaskManager(this.sinon.clock);
+      assert.isTrue(screen.mozLockOrientation.called);
```

Comment:
we should set `OrientationManager.defaultOrientation` and assert `calledWith(...)` here